src/loadtest/mock-browser.js

```diff
@@ -1,9 +1,11 @@
 import { LocalStorage } from 'node-localStorage'
-import Config from '../config/config'
 import Gun from 'gun'
 import SEA from 'gun/sea'
+// eslint-disable-next-line no-unused-vars
 import load from 'gun/lib/load'
```

Comment:
I think we can import gun load without setting a constant with that variable, so we don't need to disable eslint
```suggestion
import 'gun/lib/load'
```